packages/myst-cli/src/utils/webFileObject.ts

```diff
@@ -23,7 +22,7 @@ const FileExtensionMap: Record<string, FileExtension> = {
   [KnownCellOutputMimeTypes.AppJson]: FileExtension.json,
 };
 
-export class WebFileObject implements IFileObject {
+export class WebFileObject {
```

Comment:
Is this used elsewhere? Looks like maybe in writing base64 images? I wonder if that could be turned into a single function?

Reply:
I have removed this!